CHANGELOG.md

```diff
@@ -30,7 +30,16 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
     string after a specified substring; this is useful for pulling an user/account name from an email address. In the following example it uses the account email can be any other parameter:
     
     * `{{ truncateFrom .Account.Email "@" }}`: this would turn `foo@example.com` into `foo`
-
+* Per-Scope Key lifecycle management: Both Key Encryption Keys (KEKs) and Data Encryption
```

Comment:
I'm not sure I follow `Per-Scope Key` (is that a common term?). Does that just mean these keys are managed/organized by scope? 

EDIT: Seems like that's what it means. Maybe I was being thrown off by the capitalization in Scope? (Per-Scope -> Per-scope)

Reply:
This was fixed :)